.gitignore

```
@@ -48,3 +48,4 @@ yarn-error.log*
 
 # IDE
 .vscode/settings.json
+.idea
```

```suggestion
.idea/
```

加上 `/` 可以让开发人员了解到这是一个文件夹